bundles/org.openhab.binding.amberelectric/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="amberelectric"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="amber-electric">
```

Comment:
You could consider slightly changing the thing type id, for example to `service` or `api`, so the UID would start with e.g. "amberelectric:service" rather than "amberelectric:amber-electric".

Reply:
Done.